BaseFilter/sections/specific.txt

```diff
@@ -666,6 +666,9 @@ $third-party,script,domain=klto9.com|f2movies.to|hacamchicac.com|yourporngod.com
 ! NOTE: Specific rules
 !
 !
+gettranny.com#%#//scriptlet('abort-current-inline-script', 'AdManager')
```

Comment:
Better add a domain to https://github.com/AdguardTeam/AdguardFilters/blob/c0cd03563279c29fc48a98a04ea13d0f0ba4831b/BaseFilter/sections/general_extensions.txt#L1216 and move to BaseFilter/sections/specific.txt without a comment. It seems popupar script.